### PR TITLE
Add support for maven install

### PR DESCRIPTION
**Pull request by cheleb**
Simply allows to install artifatcts to a local repo.

### CODE REVIEW

File: maven-install.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#group is overiden to support forked repositories
+./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
```
Comment (liyinan926) [inline maven-install.sh]: Can you make the version number an input argument?

Comment (cheleb) [inline maven-install.sh]: done

On Fri, Nov 20, 2015 at 7:08 PM, Yinan Li <notifications@github.com> wrote:

> In maven-install.sh
> <https://github.com/linkedin/gobblin/pull/475#discussion_r45498650>:
>
> > @@ -0,0 +1,2 @@
> > +#group is overiden to support forked repositories
> > +./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
>
> Can you make the version number an input argument?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/gobblin/pull/475/files#r45498650>.
>



-- 
"Computers are useless. They can only give you answers."
- Pablo Picasso -

File: maven-install.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  shift
+done
+
+if [ -z "$VERSION" ]; then
+  print_usage
+  exit
+fi
+echo VERSION=$VERSION
+
+./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
```
Comment (liyinan926) [inline maven-install.sh]: Seems you are not using the arguments here.

Comment (cheleb) [inline maven-install.sh]: redone ... sorry :(
Hard end of week ...


On Fri, Nov 20, 2015 at 7:44 PM, Yinan Li <notifications@github.com> wrote:

> In maven-install.sh
> <https://github.com/linkedin/gobblin/pull/475#discussion_r45502533>:
>
> > +      print_usage
> > +      exit 0
> > +      ;;
> > +    *)
> > +      ;;
> > +  esac
> > +  shift
> > +done
> > +
> > +if [ -z "$VERSION" ]; then
> > +  print_usage
> > +  exit
> > +fi
> > +echo VERSION=$VERSION
> > +
> > +./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
>
> Seems you are not using the arguments here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/gobblin/pull/475/files#r45502533>.
>



-- 
"Computers are useless. They can only give you answers."
- Pablo Picasso -

**Comment by liyinan926** [comment]
LGTM. Thanks!

**Comment by liyinan926** [comment]
Seems the build is failing on both Java 7 and 8. Have you tested building this locally?

**Pull request merged**